ajax/libs/froala-design-blocks/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "froala-design-blocks",
+  "version": "1.0.0-alpha.1",
+  "description": "An amazing collection of over 170 ready to use design blocks based on Bootstrap.",
+  "homepage": "https://www.froala.com/design-blocks",
+  "keywords": [
+    "froala",
+    "froala design blocks",
+    "html design",
+    "bootstrap",
+    "material design",
+    "html templates",
+    "website design",
+    "html5",
+    "trending design"
+  ],
+  "license": "LGPL-2.1",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/froala/design-blocks.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/froala/design-blocks.git",
+    "fileMap": [
+      {
+        "basePath": "dist/",
+        "files": [
+          "css/froala_blocks.css"
```

Comment:
You can use the pattern `css/**/*`
Because if you generate the minified file under css folder, CDNJS bot can get the files.
Thanks.